src/languages/javascript.js

```diff
@@ -252,7 +252,7 @@ export default function(hljs) {
 
   const CLASS_REFERENCE = {
     relevance: 0,
-    match: /\b[A-Z][a-z]+([A-Z][a-z]+)*/,
+    match: /\b[A-Z][A-Za-z0-9_]+)*/,
```

Comment:
does this even work? I think you have a mismatched parenthesis.
you can remove the `+)`.

Reply:
woops! My bad! thanks for the quick review! how does it look now?